data_sha

```diff
@@ -1 +0,0 @@
-a1da8ed
```

Comment:
This will need to be reflected in:

 https://github.com/npct/pct-shiny/blob/3a06d3c6ffb42fd0ac23dfe71a50f0ce6b656b78/scripts/publish-pct-shiny

or the server will not update using the correct data.